### ISSUE TITLE
Support redmine_ckeditor plugin

### ISSUE THREAD

**Issue by popy2k14** [hdgarrood/redmine_release_notes#116]
I am using redmine_ckeditor plugin and release notes doesnt support ckeditor.

ckeditor plugin: http://www.redmine.org/plugins/ckeditor

**Comment by asinteg-daehn** [comment]
Thanks for your patch! Works great!

I applied it together with some fixes for the HTML format on my fork:
https://github.com/asinteg-daehn/redmine_release_notes

**Comment by popy2k14** [comment]
Nice that it also works for you :-)

pOpY

**Comment by asinteg-daehn** [comment]
Indeed! :+1: 

Sadly this project seems to be unmaintained currently :(

**Comment by popy2k14** [comment]
Yeah, its a great plugin. I have done several other improvements, sadly also some redmine core hacks because i dont very familiar with writing plugins which includes core patches. One improbement is an integration of the release notes in the roadmap. now i am using the roadmap as a changelog with releasenotes :-)

**Comment by asinteg-daehn** [comment]
That sounds really interesting - and I have the same problems of lacking knowledge, too.

Currently I reimplemented the display functions for the Release Notes page to support HTML better and to add a print preview dialog. In the moment I'm moving my hardcoded CSS code out into separate .css files and then all gets pushed into my open Github repo.

Your improvements sounds really really interesting - would love to see it.

Besides: I bought me a book for writing Redmine plugins (Packt Publishing) - maybe it'll help me to better understand the plugin stuff - and: maybe we could move your hacks into separate files or better: official plugin hooks. So your extensions can be part of the official plugin (or a fork of it).
